force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -80,16 +89,35 @@ public with sharing class ClassesObjectsMethods {
      * @return A string consisting of only unique words from the input sentence.
      */
     public static String removeDuplicateWords(String sentence) {
+        List<String> words = new List<String>();
+        String s;
+        
         // Split the sentence into words
+        words = sentence.split(' ');        
 
         // Create a set to store the unique words
+        Set<String> uniqueWords = new Set<String>();
 
         // Add each word to the set
+        uniqueWords.addAll(words);        
 
         // Join the unique words back into a sentence
-
+        
+        // Version 1
+        // words.clear();
+        // words.addAll(uniqueWords);
+        // s = String.join(words, ' ');
+
+        // Version 2
+        // Use Type Casting (Iterable<T>) and type cast uniqueWords SET
+        // s = String.join((Iterable<String>)uniqueWords, ' ');
+
+        // Version 3
+        // Update <apiVersion> to 58 and above
+        s = String.join(uniqueWords, ' ');
+        
         // Return the sentence with unique words
-        return null; // Replace null with the variable you used to store the result
+        return s;
```

Comment:
This is good but I think you can be a bit more concise if you want.  No need for the variable s. Just return the call the call to String.join

Here is my solution

```
  public static String removeDuplicateWords(String sentence) {
        Set<String> uniqueWords = new Set<String>(sentence.split(' '));
        return String.join(uniqueWords, ' ');
    }
    ```